docs/development/tracing.md

```diff
@@ -13,6 +13,19 @@ descriptions](https://lightstep.com/opentelemetry/tracing), and the
 [specification
 itself](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/overview.md#tracing-signal).
 
+## Quickstart
+
+1. Run [`jaeger-all-in-one`](https://www.jaegertracing.io/download/) (either from a binary, or using the docker image).
```

Comment:
We could consider expanding our `make start-dependencies` target and expand the Grafana docker setup with Tempo (Grafana's OTEL collector and UI). I do have a branch somewhere where I did that.